README.md

```diff
@@ -21,10 +21,131 @@ It implements the following architecture:
-* [Oculus SDK](https://developer.oculus.com/downloads/package/oculus-sdk-for-windows/): The native Windows SDK.
-* [Cyberith SDK](https://www.cyberith.com/research-development/): To allow the virtualizer module to capture the operator data.
+ 1. **robotology superbuild [optional]:** Install the robotology superbuild on a Windows machine as described here [Windows installation notes](https://github.com/robotology/robotology-superbuild#windows):
+
+    1. opencv: Do not use the opencv package as mentioned there (Dec 2018), instead install it from the [opencv website] (https://opencv.org/releases.html), version 3.4.4;
+
+   + if you don't want to use superbuild for installation, skip it and go directly to second dependency (YARP)
+2. **YARP:** Using superbuild install the [YARP](http://www.yarp.it/): to handle the comunication with the robot with both ovrheadset and SDLjoypad drivers;
+
+   + if you are not using superbuild, look at [this link](http://www.yarp.it/) for installing YARP.
+
+3.  **Walking-controllers:** In order to teleoperate the robot, this package should be installed in another workstation;
+
+4. In the Super-build, enable the QT5 and Eigen3 to install (follow the instructions there).
+
+5. **Oculus SDK:** In order to install the oculus SDK, yolu need to install first the Oculus app on windows (oculus application), install the depenecies(GLEW, GLFW3), and finally install the sdk (LibOVR):
+
+   1. `oculus application`: Download and install the oculus [setup application](https://www.oculus.com/setup/). It is used for both the Oculus Virtual Reality headset (ovrheadset) and joypads (touch controller).
+          
+   2. `GLEW`: Dowload the version 2.1.0 of the [glew libarary](http://glew.sourceforge.net/index.html) from this[link](https://sourceforge.net/projects/glew/files/glew/2.1.0/) (if you want to dowload from the [glew libarary](http://glew.sourceforge.net/index.html) select the source file)
+     
+      - extract and copy the glew library to your robot/code workspace (in our case in same path of robotology-superbuild)
+      
+      - the library has the makfiles, so go to the 'glew\glew-2.1.0\glew-2.1.0\build\vc12' and using the VS15 (VS 2017) build the library (release and debug!)
+      
+      - Add the following variables value in robotology/yarp using CMake GUI (search for glew!)
+      
+              `GLEW_INCLUDE_DIR`: '<path to codes workspace>/glew/glew-2.1.0/glew-2.1.0/include
+               (inside this folder you should find the GL folder and inside that the header files)
+          
+              `GLEW_LIBRARY_DEBUG`: '<path to codes workspace>/glew/glew-2.1.0/glew-2.1.0/lib/Debug/x64/glew32d.lib'
+          
+              `GLEW_LIBRARY_RELEASE`: '<path to codes workspace>/glew/glew-2.1.0/glew-2.1.0/lib/Release/x64/glew32.lib'
+              
+              `YARP_USE_GLEW`: check the box
+         
+      - Add the following values to the given variables of the Rapid Environment Editor:
```

Comment:
I think you can avoid referring to _Rapid Environment Editor_.

Reply:
I have updated it, but I kept the software we have used as an option!